modules/react/button/lib/types.ts

```diff
@@ -16,7 +16,14 @@ export interface ButtonColors {
   hover?: ButtonStateColors;
   active?: ButtonStateColors;
   focus?: ButtonStateColors & {
+    /**
+     * FocusRing within focus, is deprecated in v12 and will not work with our current method of styling.
+     * This will be removed in a later version.
+     * @deprecated
```

Comment:
```suggestion
     *  @deprecated `focusRing` within focus is deprecated and will not work with our current method of styling. This will be removed in a future major version. If you want to customize the focus ring, use `boxShadowInner` and `boxShadowOuter` to change the colors, but use this with caution.
```

Reply:
I added some additional items with that as well